core/src/main/java/org/apache/iceberg/rest/Endpoint.java

```diff
@@ -61,6 +63,19 @@ public class Endpoint {
       Endpoint.create("POST", ResourcePaths.V1_TABLE_REGISTER);
   public static final Endpoint V1_REPORT_METRICS =
       Endpoint.create("POST", ResourcePaths.V1_TABLE_METRICS);
+  public static final Endpoint V1_TABLE_EXISTS = Endpoint.create("HEAD", ResourcePaths.V1_TABLE);
+  public static final Endpoint V1_TABLE_CREDENTIALS =
```

Comment:
that endpoint is a little bit special atm because the REST client technically doesn't use it anywhere and it's only being used in the vended credential refresh handlers. I'm still ok adding this endpoint here just in order to be consistent in listing out all endpoints that are defined in the OpenAPI spec

Reply:
makes sense, I'll just leave the definition there for consistency then.